TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/TeachingRecordSystem.SupportUi.csproj

```diff
@@ -5,6 +5,10 @@
     <UserSecretsId>TeachingRecordSystemSupportUi</UserSecretsId>
   </PropertyGroup>
 
+  <Target Name="CompileSass" BeforeTargets="Build">
+    <Exec Command="sass -q wwwroot/Styles/site.scss wwwroot/Styles/site.css" />
+  </Target>
+
   <ItemGroup>
     <PackageReference Include="AspNetCore.HealthChecks.NpgSql" Version="7.0.0" />
```

Comment:
Think we need a `Microsoft.Web.LibraryManager.Build` package reference too

Reply:
Yep - didn't realise until reading libman docs - done now